README.md

```diff
@@ -264,7 +264,7 @@ Note: the `composer` command used above to train the model refers to the [Compos
 If you have a write-enabled [HuggingFace auth token](https://huggingface.co/docs/hub/security-tokens), you can optionally upload your model to the Hub! Just export your token like this:
 
 ```bash
-export HUGGING_FACE_HUB_TOKEN=your-auth-token
+export HF_TOKEN=your-auth-token
```

Comment:
which version did they deprecate in? do we need a min version bump?

other than that, lgtm

Reply:
they've supported both args for a while and still do, so no version bump needed.